plugins/inputs/sqlserver/README.md

```diff
@@ -256,31 +256,6 @@ to use them.
   ## and following as optional (if mentioned in the include_query list)
   ## - SQLServerAvailabilityReplicaStates
   ## - SQLServerDatabaseReplicaStates
-
-  ## Version 2 by default collects the following queries
-  ## Version 2 is being deprecated, please consider using database_type.
-  ## - PerformanceCounters
-  ## - WaitStatsCategorized
-  ## - DatabaseIO
-  ## - ServerProperties
-  ## - MemoryClerk
-  ## - Schedulers
-  ## - SqlRequests
-  ## - VolumeSpace
-  ## - Cpu
```

Comment:
Why do you remove this information?

Reply:
Both options are removed and are still described what they provide in the README, this doesn't need to exist in the config.